13-hybrid-programming.Rmd

```diff
@@ -0,0 +1,68 @@
+# Hybrid Programming with SAS and R
+
+This section contains current solutions of hybrid programming.
+
+## Use {sasr}
+
+{[sasr](https://github.com/insightsengineering/sasr)} is a package built on {reticulate} so that the python package {[saspy](https://github.com/sassoftware/saspy)} can be called from R.
+It adds some simple wrappers to the {saspy} functionalities to make the syntax similar to R.
+
+Here are some simple examples of how you can use it.
+```{r eval = FALSE}
+library(sasr)
+# transfer data from R to SAS
+df2sd(mtcars)
+
+# execute SAS commands
+
+run_sas("
+  data test;
+    set work.mtcars;
+  run;
+")
+
+# transfer data back to R
+
+sd2df("test")
+
+```
+This solution is based on {[saspy](https://github.com/sassoftware/saspy)}, an official python interface to connect SAS.
+This requires that you have a machine with SAS and it accessible through specific [connection types](https://sassoftware.github.io/saspy/configuration.html).
+
+## Use {jupyter}/{quarto} and {saspy}
+
+[jupyter](http://www.jupyter.org/) and [quarto](https://quarto.org/) are both capable of creating dynamic outputs from multiple languages.
+Although official SAS kernel is not supported, it can be installed through package [sas_kernel](https://github.com/sassoftware/sas_kernel) provided by [SAS software](https://github.com/sassoftware).
```

Comment:
```suggestion
Although the official SAS kernel is not supported, it can be installed through package [sas_kernel](https://github.com/sassoftware/sas_kernel) provided by [SAS software](https://github.com/sassoftware).
```